source/includes/_memory bank.md

```diff
@@ -0,0 +1,245 @@
+# Memory Bank
+
+<aside class="warning">V2 Memory Bank endpoints will be deprecated in January 2019</aside>
```

Comment:
Put deprecated in late 2019 or 2020. We don't actually have concrete plans to move off of this soon, just don't wanna be stuck on v2 forever.